lib/msal-browser/src/config/Configuration.ts

```diff
@@ -99,19 +99,11 @@ export type BrowserAuthOptions = {
      * @deprecated This flag is deprecated and will be removed in the next major version. createNestablePublicClientApplication should be used instead.
      */
     supportsNestedAppAuth?: boolean;
-    /**
-     * Callback that will be passed the url that MSAL will navigate to in redirect flows. Returning false in the callback will stop navigation.
-     */
-    onRedirectNavigate?: (url: string) => boolean | void;
 };
 
 /** @internal */
-export type InternalAuthOptions = Omit<
-    Required<BrowserAuthOptions>,
-    "onRedirectNavigate"
-> & {
+export type InternalAuthOptions = Required<BrowserAuthOptions> & {
     OIDCOptions: Required<OIDCOptions>;
-    onRedirectNavigate?: (url: string) => boolean | void;
```

Comment:
Since this released already we can't remove this. I think this was a good addition anyway even wihtout the retry

Reply:
Good point, thanks for catching that. Fixed!